app/helpers/upload_display_helper.rb

```diff
@@ -76,6 +76,8 @@ def upload_variant(model, name, variant, default: model.name.underscore)
   end
 
   def upload_default(png_name = 'profil')
+    png_name = 'profil' if png_name.include?('..')
```

Comment:
This unfortunately helps only marginally. See https://github.com/xmendez/wfuzz/blob/master/wordlist/vulns/dirTraversal-nix.txt for a big list of ideas how to do path traversal without two consecutive periods.

Reply:
Agreed. In theory, at least.

Practically, though, I cannot read even allowed files with path-traversal

`media/images/profil.png` gives me an internal path to a PNG
`media/images/../images/profile.png` gives me nothing but a Webpacker-Error
`media/images/%2e%2e/images/profile.png` gives me nothing but a Webpacker-Error

Since this is developer-provided, non-user-facing and seemingly secured already, I tend to not fix it. If a dev wants to be clever but is not, it now defaults to the default profile-picture.